doc/sphinx/language/gallina-specification-language.rst

```diff
@@ -1287,27 +1250,41 @@ constructions.
 
 .. _Fixpoint:
 
-.. cmd:: Fixpoint @ident @binders {? {struct @ident} } {? : @type } := @term
+.. cmd:: Fixpoint @fix_definition {* with @fix_definition }
+
+   .. insertprodn fix_definition fix_definition
+
+   .. prodn::
+      fix_definition ::= @ident_decl {* @binder } {? @fixannot } {? : @type } {? := @term } {? @decl_notations }
 
    This command allows defining functions by pattern matching over inductive
    objects using a fixed point construction. The meaning of this declaration is
-   to define :token:`ident` a recursive function with arguments specified by
-   the :token:`binders` such that :token:`ident` applied to arguments
-   corresponding to these :token:`binders` has type :token:`type`, and is
-   equivalent to the expression :token:`term`. The type of :token:`ident` is
-   consequently :n:`forall @binders, @type` and its value is equivalent
-   to :n:`fun @binders => @term`.
+   to define :n:`@ident` as a recursive function with arguments specified by
+   the :n:`@binder`\s such that :n:`@ident` applied to arguments
+   corresponding to these :n:`@binder`\s has type :n:`@type`, and is
+   equivalent to the expression :n:`@term`. The type of :n:`@ident` is
+   consequently :n:`forall {* @binder }, @type` and its value is equivalent
+   to :n:`fun {* @binder } => @term`.
 
    To be accepted, a :cmd:`Fixpoint` definition has to satisfy some syntactical
    constraints on a special argument called the decreasing argument. They
    are needed to ensure that the :cmd:`Fixpoint` definition always terminates.
    The point of the :n:`{struct @ident}` annotation is to let the user tell the
```

Comment:
It would be great to note the link with the `@fixannot` nonterminal in the grammar above.